delphi-frontend/src/test/java/au/com/integradev/delphi/antlr/ast/node/TextLiteralNodeImplTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Sonar Delphi Plugin
+ * Copyright (C) 2024 Integrated Application Development
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package au.com.integradev.delphi.antlr.ast.node;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import au.com.integradev.delphi.antlr.DelphiLexer;
+import org.antlr.runtime.CommonToken;
+import org.junit.jupiter.api.Test;
+import org.sonar.plugins.communitydelphi.api.ast.DelphiNode;
+
+class TextLiteralNodeImplTest {
+  @Test
+  void testMultilineImage() {
```

Comment:
It's never really tested that the grammar's handwritten multiline string parser correctly gets the contents of a multiline string.
Not sure off the top of my head how you would do that, but it's something to be aware of as a limitation.

Is there any way we can test this, even indirectly?

Reply:
The closest thing to testing that the handwritten parser collects the string properly is the multiline string tests in `GrammarTest`, but they have the meaningful limitation of (theoretically) missing cases where whitespace after the token is erroneously collected. 

It would be pretty painful/involved to do better, so I'd rather not.